scrapy_autoextract/errors.py

```diff
@@ -0,0 +1,27 @@
+from autoextract.aio.errors import DomainOccupied
+
+
+class QueryError(Exception):
+
+    def __init__(self, query: dict, message: str):
+        self.query = query
+        self.message = message
+
+    def __str__(self):
+        return f"QueryError: message='{self.message}, query={self.query}'"
+
+
+def summarize_exception(exc):
+    """
+    Provides a text that represents the exception. To be used in stats, so
+    produced text shouldn't be too diverse.
+    """
+    if isinstance(exc, QueryError):
+        msg = exc.message
+        if DomainOccupied.from_message(msg):
+            # Removes variability from domain occupy messages
+            msg = "domain occupied"
+        msg = f"/query/{msg}"
+    else:
+        msg = f"/rest/{exc.__class__.__name__}"
```

Comment:
"rest" => "request"?

Reply:
Is `rest` because is the rest of errors. It will be something like:
```
/pages/error/query: 29
/pages/error/rest: 12
```

I'm thinking that I'm going to rename `error` by `errors`

Reply:
Done here: https://github.com/scrapinghub/scrapy-autoextract/pull/16/commits/41d3426f1a02ca4f28106dac5e7c5c6d43794dee